grails-app/views/index.gsp

```diff
@@ -215,7 +215,7 @@
 
 <div class="footer">
     <div class="container">
-        <p><span class="glyphicon glyphicon-copyright-mark"></span> Gr8Ladies 2015</p>
+        <p><span class="glyphicon glyphicon-copyright-mark"></span> Gr8Ladies <script type="text/javascript">document.write(new Date().getFullYear());</script></p>
```

Comment:
Thank you so much for contributing. While you're in here, can you please update the text to be Gr8Ladies 2013 - <script... ?